### ISSUE TITLE
WCAG 1.3.1: Ensures <li> elements are used semantically (li:nth-child(2))

### ISSUE THREAD

**Issue by oklimanova** [oklimanova/hello-world#2]
#### Issue
`Ensures <li> elements are used semantically` ([listitem](https://dequeuniversity.com/rules/axe/3.3/listitem?application=msftAI))
#### Target application
[DASHBOARD - REGINFO.GOV](https://www.reginfo.gov/public/)
#### Element path
table[width="\31 00\%"] > tbody > tr:nth-child(1) > td > li:nth-child(2)
#### Snippet
`<li style="margin-left: 0; padding-top: 5px; padding-left: 25px; text-indent: -20px; font-size: 14px"><a href="javascript:show_eo(0);">Regulations under EO 12866 Review</a></li>`
#### How to fix
    Fix any of the following:
      List item does not have a <ul>, <ol> or role="list" parent element
#### Environment
Chrome version 81.0.4044.34
This accessibility issue was found using Accessibility Insights for Web 2.14.1 (axe-core 3.3.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
